src/quicklook.cs

```diff
@@ -119,6 +128,19 @@ interface QLPreviewControllerDelegate {
 		[Export ("previewController:transitionViewForPreviewItem:"), DelegateName ("QLTransitionView"), DefaultValue (null)]
 		[return: NullAllowed]
 		UIView TransitionViewForPreviewItem (QLPreviewController controller, IQLPreviewItem item);
+
+		[iOS (13,0)]
+		[Export ("previewController:editingModeForPreviewItem:"), DelegateName("QLEditingMode"), DefaultValue ("QLPreviewItemEditingMode.Disabled")]
```

Comment:
How did you know to use the DelegateName and DefaultValue attributes and their values?

Reply:
Looking at documentation and other similar code - ie, transitionViewForPreviewItem has DelegateName + DefaultValue. Then asking Alex :)